README.md

```diff
@@ -44,4 +44,4 @@ yarn dev # serves VuePress' own docs with itself
 
 ## License
 
-[MIT](https://github.com/vuejs/vuepress/blob/master/LICENSE)
```

Comment:
@ulivz Use explicit links to readme rendered by npm or docker works

Reply:
cool, updated at 46794160eddf0f6f38b5798dc612508f67231814